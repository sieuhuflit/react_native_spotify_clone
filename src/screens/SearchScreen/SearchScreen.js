import React from 'react';
import { Text, View, StyleSheet } from 'react-native';
import { colors } from '../../constants';

const styles = StyleSheet.create({
  container: {
    backgroundColor: colors.blackBg,
    flex: 1
  }
});
const SearchScreen = () => {
  return (
    <View style={styles.container}>
      <Text> SearchScreen </Text>
    </View>
  );
};

export default SearchScreen;
