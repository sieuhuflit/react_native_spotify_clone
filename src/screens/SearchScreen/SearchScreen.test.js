import React from 'react';
import { shallow } from 'enzyme';
import SearchScreen from './SearchScreen';

const setup = propOverrides => {
  const props = Object.assign({}, propOverrides);
  const wrapper = shallow(<SearchScreen {...props} />);
  return {
    props,
    wrapper
  };
};

describe('SearchScreen check snapshot', () => {
  test('should match snapshot', () => {
    const { wrapper } = setup();
    expect(wrapper).toMatchSnapshot();
  });
});
