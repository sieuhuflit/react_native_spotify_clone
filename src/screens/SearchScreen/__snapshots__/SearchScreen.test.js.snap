// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchScreen check snapshot should match snapshot 1`] = `ShallowWrapper {}`;
